src/p11_pkey.c

```diff
@@ -542,7 +542,7 @@ static int pkcs11_try_pkey_ec_sign(EVP_PKEY_CTX *evp_pkey_ctx,
 	EVP_PKEY *pkey;
 	EC_KEY *eckey;
 	PKCS11_KEY *key;
-	int rv = 0;
+	int rv = -1;
```

Comment:
What about `CKR_GENERAL_ERROR`?

Reply:
The specific value is discarded anyway, but sure, why not.